pkg/controllers/dynakube/activegate/internal/statefulset/statefulset_test.go

```diff
@@ -454,3 +454,85 @@ func TestSecurityContexts(t *testing.T) {
 		require.Truef(t, reflect.DeepEqual(sts.Spec.Template.Spec.InitContainers[0].SecurityContext, sts.Spec.Template.Spec.Containers[0].SecurityContext), "InitContainer and Container have different SecurityContexts")
 	})
 }
+
+func TestVolumes(t *testing.T) {
+	t.Run("empty dir volume exists when PersistentVolumeClaim = nil and UseEphemeralVolume = true", func(t *testing.T) {
+		dk := getTestDynakube()
+		dk.Spec.ActiveGate.PersistentVolumeClaim = nil
+		dk.Spec.ActiveGate.UseEphemeralVolume = true
+		multiCapability := capability.NewMultiCapability(&dk)
+		statefulsetBuilder := NewStatefulSetBuilder(testKubeUID, testConfigHash, dk, multiCapability)
+		sts, _ := statefulsetBuilder.CreateStatefulSet([]builder.Modifier{
+			modifiers.NewKubernetesMonitoringModifier(dk, multiCapability),
+			modifiers.NewReadOnlyModifier(dk),
+		})
+
+		require.NotEmpty(t, sts)
+
+		expectedVolume := corev1.Volume{
+			Name: consts.GatewayTmpVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		}
+		require.Contains(t, sts.Spec.Template.Spec.Volumes, expectedVolume)
+	})
+}
+
+func TestVolumeMounts(t *testing.T) {
+	t.Run("default OTLP ingest volume mount is presented in Container volumeMount list", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("volume mount is presented in Container volumeMount list", func(t *testing.T) {
```

it's always the same volume mount regardless of the current configuration. Maybe similar test for EphemeralVolume could be added?

Reply:
fixed in 98d85a5a4